docs/standard_client/usage.rst

```diff
@@ -1,10 +1,15 @@
-Using the Sync Client
-=====================
+Standard Client
```

Comment:
Is "Standard Client" a good name for this?

Reply:
Potentially? Not sure what's a good name for it though. `Bigtable client` might be a more generically encapsulating name, but might be confusing with the API section right below it

Reply:
Maybe "Classic Client"? There's the complication that we also have to communicate the differences between the admin api vs the data api, so API is a bit overloaded

Reply:
Classic client seems good